pkg/api/types.go

```diff
@@ -98,6 +99,7 @@ func ProtoReplicaToReplica(r *spdkrpc.Replica) *Replica {
 		State:      r.State,
 		ErrorMsg:   r.ErrorMsg,
 		Rebuilding: r.Rebuilding,
+		UUID:       r.Uuid,
```

Comment:
What is the difference between LvsUUID and UUID?

Reply:
LvsUUID is the UUID of the logical volume store (LVS). UUID is for the logical volume (LV). LVS is a storage pool for LVs.